test/common/shared.spells.test.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable camelcase */
+let shared = require('../../common/script/index.js');
+
+let newUser = (addTasks = true) => {
+  let buffs;
+  let user;
+
+  buffs = {
+    per: 0,
+    int: 0,
+    con: 0,
+    str: 0,
+    stealth: 0,
+    streaks: false,
+  };
+  user = {
+    auth: {
+      timestamps: {},
+    },
+    stats: {
+      str: 1,
+      con: 1,
+      per: 1,
+      int: 1,
+      mp: 32,
+      lvl: 1,
+      class: 'rogue',
+      buffs,
+    },
+    items: {
+      lastDrop: {
+        count: 0,
+      },
+      hatchingPotions: {},
+      eggs: {},
+      food: {},
+      gear: {
+        equipped: {},
+        costume: {},
+      },
+      quests: {},
+    },
+    party: {
+      quest: {
+        progress: {
+          down: 0,
+        },
+      },
+    },
+    preferences: {},
+    dailys: [],
+    todos: [],
+    rewards: [],
+    flags: {},
+    achievements: {},
+    contributor: {
+      level: 100,
+    },
+  };
+  shared.wrap(user);
+  user.ops.reset(null, () => {});
+  if (addTasks) {
+    _.each(['habit', 'todo', 'daily'], (task) => {
+      user.ops.addTask({
+        body: {
+          type: task,
+          id: shared.uuid(),
+        },
+      });
+    });
+  }
+  return user;
+};
+
+describe('Spells', () => {
+  let user;
+
+  beforeEach(() => {
+    user = newUser();
+    shared.wrap(user);
+  });
+
+  context('Rogue Spells', () => {
+    it('Should add exp and gp to user when backstab is used', () => {
+      let previousExp = user.stats.exp;
+      let previousGp = user.stats.gp;
+
+      shared.content.spells.rogue.backStab.cast(user, user.todos[0]);
+      expect(user.stats.exp).to.greaterThan(previousExp);
+      expect(user.stats.gp).to.greaterThan(previousGp);
+    });
+
+    it('Should pass when user is leveled passed all breaks and backstab is used', () => {
```

Comment:
I don't understand this description. "It should pass" - sure, all tests should pass, but what's actually being tested? What does "user is leveled passed all breaks" mean?